server/public/style.css

```diff
@@ -2,3 +2,31 @@
   display: flex;
   justify-content: space-between;
 }
+
+.container{
+  display: flex;
+  flex-wrap: wrap;
+
+}
```

Comment:
Bootstrap already has a `container` class. Perhaps some of their `flex` helper classes are what you're after.